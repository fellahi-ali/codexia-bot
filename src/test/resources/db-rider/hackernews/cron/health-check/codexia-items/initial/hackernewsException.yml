github_repo:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 202275988
        full_name: casbin/casbin
        html_url: https://github.com/casbin/casbin
        repo_created_at: 2019-08-14 04:42:36

github_repo_source:
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        source: "HACKERNEWS"
        external_id: "2222"
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        source: "CODEXIA"
        external_id: "666"

hackernews_item:
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 2222
        type: story
        by: smartmic
        title: Title for casbin
        url: https://github.com/casbin/casbin
        time: 2020-02-14 18:43:33
        deleted: false
