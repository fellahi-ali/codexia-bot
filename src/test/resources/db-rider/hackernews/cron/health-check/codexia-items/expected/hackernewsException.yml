github_repo_source:
    -
        github_repo_id: 1
        source: "HACKERNEWS"
        external_id: "2222"
        deleted_at: null
    -
        github_repo_id: 1
        source: "CODEXIA"
        external_id: "666"
        deleted_at: null

hackernews_item:
    -
        external_id: 2222
        type: story
        by: smartmic
        title: Title for casbin
        url: https://github.com/casbin/casbin
        time: 2020-02-14 18:43:33
        deleted: false
